packages/flutter_tools/test/general.shard/flutter_tester_device_test.dart

```diff
@@ -216,15 +227,17 @@ class TestFlutterTesterDevice extends FlutterTesterTestDevice {
     compileExpression: null,
     fontConfigManager: FontConfigManager(),
   );
+  late DartDevelopmentService dds;
 
   final Completer<Uri> _ddsServiceUriCompleter = Completer<Uri>();
 
   Future<Uri> ddsServiceUriFuture() => _ddsServiceUriCompleter.future;
 
   @override
-  Future<DartDevelopmentService> startDds(Uri uri) async {
+  Future<DartDevelopmentService> startDds(Uri uri, {UriConverter? uriConverter}) async {
     _ddsServiceUriCompleter.complete(uri);
-    return FakeDartDevelopmentService(Uri.parse('http://localhost:${debuggingOptions.hostVmServicePort}'), Uri.parse('http://localhost:8080'));
+    dds = FakeDartDevelopmentService(Uri.parse('http://localhost:${debuggingOptions.hostVmServicePort}'), Uri.parse('http://localhost:8080'), uriConverter: uriConverter);
```

Comment:
nit: Similarly here, I think breaking this over multiple lines using trailing commas would help in readability too